### ISSUE TITLE
polymer init (2.0 element) sets polymer-2-hybrid as default lint rule instead of polymer-2

### ISSUE THREAD

**Issue by peterlauri** [Polymer/polymer-cli#736]
### Description
When running **polymer init** and creating a Polymer 2 element, the default line rule is set to polymer-2-hybrid and not polymer-2.

### Versions & Environment
- Polymer CLI: 0.18.3
- node: v6.10.1
- Operating System: OSX

#### Steps to Reproduce
1. Install polymer-cli==0.18.3
2. run **polymer init** and select the second option (Polymer 2 element)
3. run **cat polymer.json**

#### Expected Results
```json
{
  "lint": {
    "rules": [
      "polymer-2"
    ]
  }
}
```

#### Actual Results
```json
{
  "lint": {
    "rules": [
      "polymer-2-hybrid"
    ]
  }
}
```

**Comment by FredKSchott** [comment]
@peterlauri doh! Thanks for catching that. Any interest in pushing a fix?

**Comment by abdonrd** [comment]
https://github.com/Polymer/polymer-cli/pull/772

**Comment by FredKSchott** [comment]
Fixed by #772.
